src/api/app/controllers/webui/package_controller.rb

```diff
@@ -241,28 +223,6 @@ def revisions
     switch_to_webui2
   end
 
-  # TODO: bento_only
-  def submit_request_dialog
```

Comment:
There are still specs for this action in [spec/controllers/webui/package_controller_spec.rb](https://github.com/openSUSE/open-build-service/blob/69274b51f0b11298cdd7db49852fd14e1f769622/src/api/spec/controllers/webui/package_controller_spec.rb#L1360)

Reply:
@dmarcoux thanks and done :+1: